src/types.ts

```diff
@@ -385,6 +385,7 @@ export interface Post {
 	inductee_address?: string;
 	typeOfReferendum?: EReferendumType;
 	allowedCommentors?: EAllowedCommentor[];
+	progress_report?: any;
```

Comment:
The type should not be any, it should be a string, since we only send a URL to the file from BE